### ISSUE TITLE
(Samsung) Can't hide Navbar anymore

### ISSUE THREAD

**Issue by ghost** [Deletescape-Media/Lawnchair#1184]
## Expected Behavior

Going to settings or doubletapping the navbar-remove button will make the navbar invisible.

## Actual Behavior

Samsungs ROM refuses to make the Navbar invisible, option in setting is greyed out and navbar-remove button doesnt exist for the launcher.

## Steps to Reproduce

1. Open the launcher

## Environment

* Device: Samsung Note 8
* Android version: 7.1.1
* Launcher version: 1.1.0.1864
* Rom: Stock

Little note: This is a version specific problem, i tested a few other versions, all of which work as expected

**Comment by divadsn** [comment]
1.1.0.1864 isn't the latest one.

**Comment by Nevexo** [comment]
Can reproduce on Oreo / galaxy S8 version 1868

**Comment by divadsn** [comment]
@Nevexo It is intentional afaik, @fonix232 any comment on that?

**Comment by ghost** [comment]
I meant 1868 initially, rip sorry for that

**Comment by ghost** [comment]
@divadsn Anything we can do against it? 1872 didnt fix it sadly

**Comment by divadsn** [comment]
@BeatPlay It's intended behavior dude, we've added the metadata which is the same on other launchers like Nova. I don't have a Samsung device, but @fonix232 can clear this up.

**Comment by ghost** [comment]
thanks anyways, good old ADB saved my day tho, thats why I love Anroid lol. "adb shell settings put global policy_control immersive.navigation=*" inverts it back to hidden. 

And if it's like Nova it should be able to hide, just tested it and Nova allows it!

**Comment by divadsn** [comment]
@BeatPlay Nova uses hackery to access the APIs, which will be discouraged soon on the Play Store if such behavior is detected.

**Comment by fonix232** [comment]
@BeatPlay Samsung used the meta tag we added with @divadsn a few weeks ago, on Nougat. I checked on Oreo and it is indeed not working, most likely Samsung changed something.

Nova achieves this with hackery, yes, forcing immersive mode onto the app. While I understand that for many this is an aching point, I don't think we should introduce hardcoded workarounds for platform-specific issues. 

So the answer would be, we won't fix this, as this is the intended behaviour. If a meta tag can fix it, then we can add it, but I myself won't add platform or OEM specific workarounds, and I highly discourage @deletescape @divadsn and @paphonb from doing so too.
